### ISSUE TITLE
jfrog rt bce or bag adds an unnecessary module id to a build info

### ISSUE THREAD

**Issue by tsuyo** [jfrog/jfrog-cli#1349]
**Describe the bug**
`jfrog rt bce` or `jfrog rt bag` commands after `jfrog mvn install` adds an unnecessary module id to a build info.

**To Reproduce**
```
$ jfrog mvn clean install --project=hello --build-name=hello-maven-manual-build --build-number=7
$ jfrog rt bce --project=hello hello-maven-manual-build 7
$ jfrog rt bag --project=hello hello-maven-manual-build 7 ..
$ jfrog rt bp --project=hello hello-maven-manual-build 7
```

**Expected behavior**
The following entry should not be included in the final build info
```
...
    {
      "properties": {},
      "id": "hello-maven-manual-build"
    },
...
```

**Screenshots**
N/A

**Versions**
- JFrog CLI version: 2.9.0
- JFrog CLI operating system: MacOS Monterey 12.0.1
- Artifactory Version: Not related to Artifactory nor its version

**Additional context**
N/A

**Comment by RobiNino** [comment]
Hi @tsuyo ,
Thank you for reporting this.
JFrog CLI version 2.12.0 was just released and includes a fix for this issue.
We'd appreciate your feedback for it.

**Comment by tsuyo** [comment]
Hi @RobiNino ,
I have confirmed this fix with 2.12.0 as well. Thanks for your dispatch!

**Comment by RobiNino** [comment]
Thank you for your feedback @tsuyo !
